### DISCUSSION TITLE
SplitView Pane does not render

### DISCUSSION THREAD

**Discussion by Timo-Weike** [AvaloniaUI/Avalonia#17224] [Q&A]
### Describe the bug

I started a new avalonia project and wanted to use the `SplitView` like I did in a previous project that used version `0.10.18`. The new project is now on avalonia `11.1.3`. However the SplitView does not render the side pane. It looks like the Style for the SplitView does not include the Pane as an element.

This is the visual tree in my old project
![grafik](https://github.com/user-attachments/assets/48f7e6e0-b0af-44e9-925e-057a21fb9550)

This is the visual tree of my new project
![grafik](https://github.com/user-attachments/assets/bda34847-c31e-4132-85a3-7502bf350405)

### To Reproduce

Checkout my project [repo](https://github.com/Timo-Weike/catser) and see that the pane does not render.

### Avalonia version

11.1.3

### OS

Windows

**Comment by stevemonaco** [comment]
This project appears to have been migrated from 0.10.x. Please read the migration guide: https://docs.avaloniaui.net/docs/stay-up-to-date/upgrade-from-0.10

Theming has changed and you need to explicitly include a theme now as Avalonia no longer ships with one. New project templates reflect this. No theme = no visuals.

**Reply by maxkatz6** [reply]
Yes, new app xaml doesn't include any control styles. Like, FluentTheme or third party.

![image](https://github.com/user-attachments/assets/25480579-f85f-48df-b15a-d4ea2f39c4e1)
